test/player_integration.js

```diff
@@ -344,6 +344,47 @@ describe('Player', () => {
       expect(cues.length).toBeGreaterThan(0);
     });
 
+    it('appends thumbnails for external thumbnails with sprites', async () => {
```

Comment:
But it _doesn't_ do that, right?  It just skips them right now?

Reply:
Fixed